galaxy/tools/verify/interactor.py

```diff
@@ -287,29 +326,28 @@ def stage_data_async(self, test_data, history_id, tool_id, force_path_paste=Fals
                         "files_%d|url_paste" % i: "file://" + file_name
```

Comment:
This is still broken at line 323 because of the removal of the `test_data_path()` method, not sure what's the best way to resolve this conflict. Ping @mvdbeek , there is a conflict between the changes introduced in 19.01 and https://github.com/galaxyproject/galaxy-lib/pull/137 .